app/views/organizations/staff/custom_form/questions/_form.html.erb

```diff
@@ -1,6 +1,9 @@
 <div class="row">
   <div class="col-md-8 mx-auto card p-5 bigger">
-    <%= bootstrap_form_with model: [:staff, form, question] do |f| %>
+    <%= bootstrap_form_with(
```

Comment:
Just so I understand, the new namespacing required you to now explicitly provide the URL?

Reply:
Yes, if we don't mention URL it throws issue with route path.